getting-started/pull-request-lifecycle.rst

```diff
@@ -180,7 +187,14 @@ Making Good PRs
 When creating a pull request for submission, there are several things that you
 should do to help ensure that your pull request is accepted.
 
-First, make sure to follow Python's style guidelines. For Python code you
+First, make your change against the right version of Python. Even if you're
+fixing a bug, your fix needs to go to the ``main`` branch first. After the
+fix is merged, it will be :ref:`ported back <branch-merge>` to older
+maintenance releases as well. That way we ensure all affected versions are
+handled. Unless you're sure that your change does not apply to ``main``,
+don't base your fixes on old versions of Python.
```

Comment:
The tone seems to assume that they are going to do it against the wrong version.  I would use a more neutral tone and state that almost all changes (including bug and security fixes) should be done against `main`, but in certain cases they should be done on older branches (maybe list a couple of examples (IIRC they are already mentioned somewhere)).

Providing the background for this (e.g. by explaining that we backport fixes like you did) and cases where the PR should be done on old branches, will help reassure the reader about which versions will receive the fix.